Userland/Utilities/test-fuzz.cpp

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (c) 2021, Ben Wiederhake <BenWiederhake.GitHub@gmx.de>
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#include <AK/String.h>
+#include <LibCore/ArgsParser.h>
+#include <LibCore/File.h>
+#include <stdio.h>
+#include <unistd.h>
+
+#define ENUMERATE_TARGETS(T) \
+    T(BMPLoader)             \
+    T(ELF)                   \
+    T(Gemini)                \
+    T(GIFLoader)             \
+    T(HttpRequest)           \
+    T(ICOLoader)
+/*
+//    T(JPGLoader) \
+//    T(Js) \
+//    T(Markdown) \
+//    T(PBMLoader) \
+//    T(PGMLoader) \
+//    T(PNGLoader) \
+//    T(PPMLoader) \
+//    T(RegexECMA262) \
+//    T(RegexPosixExtended) \
+//    T(Shell) \
+//    T(TTF) \
+//    T(URL)
+*/
```

Comment:
Any particular reason why these are disabled?

Reply:
Nope, just laziness.

Reply:
Converted to draft until I finish that part.

Reply:
No real reason to block on that, just wondering if there was some issue with those specific fuzzers.